src/app/Pages/SearchPage/index.jsx

```diff
@@ -79,6 +118,112 @@ export default class SearchPage extends Component {
             </button>
           </div>
         </form>
+
```

Comment:
I think we should only show this for housing officers for now since this whole area of functionality is moving rapidly and we don't want to confuse existing users. Here's how we do this elsewhere:
https://github.com/LBHackney-IT/housing-needs-single-view-ui/pull/85/files#diff-6e7226c865ede58997f5300d5ba7d8bbR135